### ISSUE TITLE
SEO data

### ISSUE THREAD

**Issue by prokaktus** [ellmetha/django-machina#124]
Hello!

What do you think about adding additional SEO data to topics?
![2018-06-12_15-18-38](https://user-images.githubusercontent.com/193254/41289972-131348d0-6e54-11e8-83b9-6c6f9510728e.png)

Now description is empty and OG-data missing.

Thanks a lot!

**Comment by ellmetha** [comment]
It could be a great idea. I won't have time to work on such an improvement anytime soon; would you like to tackle it and submit a PR? 😃

**Comment by herjanp** [comment]
Yes! you can add new SEO data on website. Its not a problem. Of course its good idea. 

Thanks

**Comment by josylad** [comment]
I hope this will be looked into. 
SEO is very important, it will be nice to use excerpts from forum posts/topics as meta descriptions and the likes.
